cfspopcon/key_checker.py

```diff
@@ -0,0 +1,195 @@
+"""A simple tool to ensure consistency between the algorithms, default units and the physics glossary, with a CLI."""
```

Comment:
I think we should move this script outside `cfspopcon` as a helper script which we can then run in CI to ensure things won't break. 

We could put it into a `tests/utils/` folder for example.